enable/examples/demo/enable/tools/pyface/context_menu.py

```diff
@@ -34,7 +34,9 @@ def _create_window(self):
         box.tools.append(context_menu)
         container = Container(bounds=[500,500])
         container.add(box)
-        return Window(self, -1, component=container)
+
+        return container
+
 
 if __name__ == "__main__":
-    demo_main(MyFrame)
+    demo_main(Demo)
```

Comment:
This example still doesn't seem to be working as it is supposed to (I'm not seeing a context menu).  These changes just allow it to run and run in `etsdemo` in its current state